pkg/plugin/plugin_test.go

```diff
@@ -41,7 +41,8 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
-	exec.Command("ovs-vsctl", "del-br", "--if-exists", BRIDGE_NAME).Run()
+	exec.Command("ovs-vsctl", "del-br", BRIDGE_NAME).Run()
+	exec.Command("ovs-vsctl", "del-br", "br-int").Run()
```

Comment:
We should not do this. BRIDGE_NAME is a "test-bridge", it's unlikely we would break something for user. However, with this we might accidentally remove real "br-int" bridge.

If we move the defaulting into a separate function, we might test that function and don't actually remove anything.

Reply:
I can't understand this comment can you please explain me in more details?

Reply:
Sure. So the problem could be: I run those tests directly on my laptop - with OVS running. This tests creates BRIDGE_NAME bridge and then removes it, that does not do any harm, since "test-bridge" is unlikely to be used for anything but tests. On the other hand, if I had OVN running on my machine, this test would use it and once it is done, it would remove it. What if "br-int" was
actually used there?

So what I suggest is. Move defaulting (if ovsPort is set and no bridge provided, set bridge to br-int) into a separate funtion. Then we can have a unit test for this function.